tutorials/math/interpolation.rst

```diff
@@ -138,4 +142,48 @@ Here is how it looks:
 
 .. image:: img/interpolation_follow.gif
 
-This is useful for smoothing camera movement, allies following you (ensuring they stay within a certain range), and many other common game patterns.
+This is useful for smoothing camera movement, for allies following the player
+(ensuring they stay within a certain range), and for many other common game patterns.
+
+.. note::
+ Despite using ``delta``, the formula used above is framerate-dependent, because
+ the ``weight`` parameter of ``lerp()`` represents a *percentage* of the remaining
+ difference in values, not an *absolute amount to change*. In ``_physics_process()``,
```

Comment:
If it's not clear from a single sentence, explaining why `weight` being a percentage rather than an absolute amount matters feels beyond the scope of the docs. There are two good links to get a deeper explanation, if the reader wants them.

Reply:
I think this is fine.